frontend/src/model/beta/ModelImages.tsx

```diff
@@ -21,8 +21,8 @@ export default function ModelImages({ model }: AccessRequestsProps) {
   const modelImageList = useMemo(
     () =>
       modelImages.length ? (
-        modelImages.map((modelImage) => (
-          <ModelImageDisplay modelImage={modelImage} key={`${modelImage.namespace}-${modelImage.model}`} />
+        modelImages.map((modelImage, index) => (
```

Comment:
Index isn't a stable key. See: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Specifically:
"The key is used by React to [identify which items have changed, are added, or are removed and should be stable](https://facebook.github.io/react/docs/lists-and-keys.html#keys).

It's a bad idea to use the array index since it doesn't uniquely identify your elements. In cases where the array is sorted or an element is added to the beginning of the array, the index will be changed even though the element representing that index may be the same. This results in unnecessary renders."

Reply:
Assuming these images have the same properties, could you use the same key as suggested in the following comment? https://github.com/gchq/Bailo/pull/823/files/709b0a4d477c71508a2394e0b5922a2b6d1a1880..12dc11db8adf0e8136f6f7d2906a5f82c265e584?diff=unified&w=1#r1377668028